docs/tutorials/upgrading-airbyte.md

```diff
@@ -44,7 +44,7 @@ If you inadvertently upgrade to a version of Airbyte that is not compatible with
    Here's an example of what it might look like with the values filled in. It assumes that the downloaded `airbyte_archive.tar.gz` is in `/tmp`.
    
    ```bash
-   docker run --rm -v /tmp:/config airbyte/migration:0.21.1-alpha --\
+   docker run --rm -v /tmp:/config airbyte/migration:0.22.0-alpha --\
```

Comment:
is this command working if you didn't create the migration class for that version though?

(Shouldn't we do patch bump versions if no migrations are included?)

Reply:
argh thanks for the reminder, I will create one now.